authors/apps/comments/views.py

```diff
@@ -0,0 +1,137 @@
+from rest_framework.generics import (CreateAPIView, RetrieveAPIView,
+                                     ListAPIView, RetrieveUpdateDestroyAPIView)
+from rest_framework.response import Response
+from rest_framework import status
+from .models import Comments
+from ..articles.models import Articles
+from .serializers import (CommentsSerializer, CommentHistorySerializer)
+from rest_framework.permissions import IsAuthenticated
+from django.shortcuts import get_object_or_404
+
+
+class CreateCommentView(CreateAPIView, ListAPIView):
+    '''Endpoint for creating a comments'''
+    queryset = Comments.objects.all()
+    serializer_class = CommentsSerializer
+    lookup_field = 'article_slug'
+    permission_classes = (IsAuthenticated,)
+
+    def post(self, request, article_slug):
+        '''Handles all post requests to create a comment'''
+
+        if Articles.objects.filter(slug=article_slug).exists():
+            # check if user is authorized
+            comment = request.data.get('comment', {})
+            article = Articles.objects.filter(slug=article_slug).first()
+            comment.update({'article': article})
+            comment.update({'user': request.user.id})
```

Comment:
What do you think about removing this one line since we are also reassigning the `user` prop below as

```py
comment['user'] = request.user
```